Extensions/Spine/JsExtension.js

```diff
@@ -73,6 +73,42 @@ module.exports = {
       .setFunctionName('setAnimationMixingDuration')
       .setGetter('getAnimationMixingDuration');
 
+    object
+      .addExpressionAndCondition(
+        'number',
+        'PointAttachmentX',
+        _('Point attachment x'),
+        _('x position of spine point attachment'),
+        _(
+          'x position of spine _PARAM1_ point attachment for _PARAM2_ slot (use "" if names are the same)'
```

Comment:
> (use "" if names are the same)

This shouldn't be visible in the eventsheet, it can be moved in the description, or better like you did in the description of the parameter.

Reply:
done